requirements-docs.in

```diff
@@ -48,6 +48,6 @@ xmltodict
 -r requirements-tests.txt
 
 # docs specific
-sphinx < 4.0 # related to https://github.com/repoze/repoze.sphinx.autointerface/pull/17
+sphinx < 5.0 # related to https://github.com/repoze/repoze.sphinx.autointerface/pull/17
```

Comment:
Looks like we're actually okay to upgrade this now:

> We got a temporary reprieve in sphinx#9326 but it will come back in Sphinx 5.0.

https://github.com/repoze/repoze.sphinx.autointerface/pull/17#issuecomment-860250733

Reply:
nice